sycl/test-e2e/Matrix/joint_matrix_down_convert_impl.hpp

```diff
@@ -0,0 +1,165 @@
+#define TM 8
+#define TN SG_SZ
```

Comment:
This test will not work if TK != TN, so initializing TN with subgroup size may cause errors.